src/robot_interface/telemetry/payloads.py

```diff
@@ -16,3 +19,25 @@ class TelemetryBatteryPayload:
     battery_level: float
     robot_id: str
     timestamp: datetime
+
+
+@dataclass
+class VideoStream:
+    name: str
+    url: str
```

Comment:
Might want to move this to the get information endpoint, so we dont have to publish this all the time

Reply:
Ended up with a solution where the info is split into two different topics such that it is a bit more natural.